tests/StartupAPITests.js

```diff
@@ -31,29 +30,34 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
         }
     });
 
-    gpii.startupAPI.tests.ConfigName = function () {
+    gpii.startupAPI.tests.DefaultConfigName = function () {
```

Comment:
May as well normalise the case of all members of this namespace following the somewhat accepted convention that initial capital letters indicate constructors

Reply:
Done, and that'll be our fourth CI run.